src/index.test.ts

```diff
@@ -431,4 +431,21 @@ describe('ember-template-recast', function () {
       '{{foo-bar placeholder="Choose a \\"thing\\"..." p1="Choose a \\"thing\\"..."}}'
     );
   });
+
+  test('stack trace does not exceed', function () {
+    let b = builders;
+    let template = '<div @class="{{if foo "bar"}} baz" />';
+    let { code } = transform({
+      template,
+      plugin() {
+        return {
+          AttrNode(node) {
+            node.value = b.concat([b.text('foobar')]);
```

Comment:
This isn't a great example (this doesn't need to be a concat statement at all), can you have it  include multiple items? Maybe:

```suggestion
            node.value = b.concat([b.text('foobar'), b.text('baz')]);
```